### PR TITLE
Initiate a branch to create tests.

### PR DESCRIPTION
**Pull request by prisae**
First make plain version and html version the same by removing `Date :` from the plain text version.

### CODE REVIEW

**Comment by prisae** [comment]
A PR to start writing tests, see #20

**Comment by prisae** [comment]
New tests are currently not yet implemented with Travis, but can be locally run with
```
pytest --cov=scooby
```

**Comment by prisae** [comment]
These are very basic/simple tests, but a start.
- To be improved/extended.
- To be integrated with Travis.

**Comment by prisae** [comment]
I had some tests here before switching to `scooby`; https://github.com/empymod/emg3d/commit/88f73115b1c9b8682b4555ee43040a613cd7b21a?diff=split#diff-87010571bbeeee0b9a13279c493670f3

File: scooby/report.py [hunk 1]
```diff
@@ -176,13 +176,9 @@ def __repr__(self):
...
-        date_text = '  Date: '
-        mult = 0
-        indent = len(date_text)
-        for txt in textwrap.wrap(self.date, self.text_width-indent):
-            date_text += ' '*mult + txt + '\n'
-            mult = indent
-        text += date_text+'\n'
+        for txt in textwrap.wrap(self.date, self.text_width-4):
+            text += '  ' + txt + '\n'
+        text += '\n'
```
Comment (banesullivan) [inline scooby/report.py]: This is the only thing that wasn't recently implemented on the master branch

**Comment by prisae** [comment]
I am closing this, Bane included these tests at some point.

**Pull request closed without merging**